scorep/__main__.py

```diff
@@ -11,6 +12,39 @@ def _err_exit(msg):
     sys.exit(1)
 
 
+def print_help():
+    print("""
+Usage: python -m scorep [options] [--] your_program.py [args]
+
+Score-P Python instrumentation wrapper. The following options control how the program is instrumented and executed:
+
+  --help                   Show this help message and exit.
+  --mpi                    Enable MPI instrumentation (equivalent to --mpp=mpi).
+  --keep-files             Keep temporary files after execution.
+  --verbose, -v            Enable verbose output for debugging and tracing.
+  --nocompiler             Disable compiler-based instrumentation.
+  --nopython               Disable instrumentation of Python code.
+                           Instrumentation can still be enabled later from within the application's source code.
+  --noinstrumenter         Same as --nopython.
+  --instrumenter-type=<t>  Specify custom instrumenter type (e.g., cProfile).
+  --instrumenter-file=<f>  Path to a Python script that is executed before the application.
+                           Allows instrumentation of specific modules and functions without modifying their source code.
+  --                       Stop parsing Score-P options; pass following args to your script.
```

Comment:
```suggestion
  --                       Stop parsing Score-P options; pass following args to your script.
```
that could be understood that `--foo -- --baz ./bar --arg` will be `./bar --arg --baz`

"; interpret all following arguments verbatim as the program with its arguments"